README.md

```diff
@@ -7,6 +15,79 @@
 | Last release                     | [![Anaconda-Server Badge](https://anaconda.org/pyomeca/pyomeca/badges/latest_release_date.svg)](https://anaconda.org/pyomeca/pyomeca)                                                                                                                        |
 | OS support                       | [![Anaconda-Server Badge](https://anaconda.org/pyomeca/pyomeca/badges/platforms.svg)](https://anaconda.org/pyomeca/pyomeca)                                                                                                                                  |
 
-![Pyomeca](https://github.com/pyomeca/design/blob/master/logo/logo_plain.svg)
+## Pyomeca documentation
+
+See pyomeca's [documentation site]() (under construction and not usable yet), as well as pyomeca's [tutorial notebooks](https://github.com/pyomeca/tutorials).
+
+## Example
+
+Here is an example of a complete EMG pipeline in just one command:
+
+```python
+from pyomeca import Analogs3d
+
+emg = (
+    Analogs3d.from_c3d("path/to/your/c3d.c3d", names=['anterior_deltoid', 'biceps'])
+    .band_pass(freq=2000, order=4, cutoff=[10, 425])
+    .center()
+    .rectify()
+    .low_pass(freq=2000, order=4, cutoff=5)
+    .normalization()
+    .time_normalization()
+)
+```
+
+## Features
+
+- Object-oriented architecture where each class is associated with common and specialized functionalities:
+  - **Markers3d**: 3d markers positions
+  - **Analogs3d**: analogs (emg, force or any analog signal)
+  - **GeneralizedCoordinate**: generalized coordinate (joint angle)
+  - **RotoTrans**: roto-translation matrix
+
+
+- Specialized functionalities include signal processing routine commonly used in biomechanics: filters, normalization, onset detection, outliers detection, derivative, etc.
+
+
+- Each functionality can be chained. In addition to making it easier to write and read code, it allows you to add and remove analysis steps easily (such as Lego blocks).
+
+
+- Each class inherits from a numpy array, so you can create your own analysis step easily.
+
+
+- Easy reading and writing interface to common files in biomechanics:
+  - **c3d**: `from_c3d` and `to_c3d`
+  - **csv**: `from_csv` and `to_csv`
+  - **mat**: `from_mat` and `to_mat`
+  - **sto**: `to_sto` (must install pyosim)
+  - **trc**: `to_trc` (must install pyosim)
+
+
+- Common linear algebra routine implemented: get Euler angles to/from roto-translation matrix, create a system of axes, set a rotation or translation, transpose or inverse, etc.
+
+## Installation
```

Comment:
I don't think we support anything other than conda at the moment, as ezc3d doesn't have a setup.py.

Reply:
oh, I see what you mean..

And assuming an advance user which can independently install the dependency, is he able to install pyomeca without conda? I kind of remember that the setup file could do the job, but I am not sure

Reply:
pyomeca itself does not require conda (`python install setup.py` is enough); but it is the dependencies that need it (ezc3d in this case).

Reply:
Yep that is what I mean, I think the Python command to install should also figure in the installation's method along with the dependency one should install in order to make pyomeca run. 

The main reason for that is, for example, when I install it myself, most of the time I can't use conda because the python version (or one of the packages) doesn't match the one I need thanks to another project I am working on (e.g. pygmo). This is not probably required by many people, but I think, as a reminder, it could be nice to add that manual installation is also a possibility. 

That said, if you think this is an edge case, I quite agree also. Therefore, I will accept the review now, but won't merge myself. You can then merge if you are satisfied the way it is already or make the modifications if you feel like so :)

Reply:
You made a good point. I have added the following instructions:

> ### Using pip
> First, you need to install python, swig, numpy & pyyaml. 
> Then, follow the instructions to compile [ezc3d](https://github.com/pyomeca/ezc3d).
> Finally, install pyomeca with `pip install git+https://github.com/pyomeca/pyomeca/`.

Unfortunately, `pip` does not detect that ezc3d is already installed and looks in the pypi repositories:

> `Could not find a version that satisfies the requirement ezc3d (from pyomeca===initial-release-37.g40d7b55) (from versions: )
No matching distribution found for ezc3d (from pyomeca===initial-release-37.g40d7b55)`

So there are two problems with ezc3d at the moment:
- does not work on windows (_c.f._,[appveyor](https://ci.appveyor.com/project/romainmartinez/pyomeca/builds/20338708))
- cannot be installed with pip (a `setup.py` in the repo would solve the problem)